docker-compose/start_dse69_dev_mode.sh

```diff
@@ -10,14 +10,13 @@ fi
 LOGLEVEL=INFO
 
 # Default to images used in project integration tests
-DSETAG="$(../mvnw -f .. help:evaluate -Dexpression=stargate.int-test.cassandra.image-tag -q -DforceStdout)"
-SGTAG="$(../mvnw -f .. help:evaluate -Dexpression=stargate.int-test.coordinator.image-tag -q -DforceStdout)"
+DSETAG="$(../mvnw -f .. help:evaluate -Pdse69-it -Dexpression=stargate.int-test.cassandra.image-tag -q -DforceStdout)"
```

Comment:
The concept of a "dev mode" docker compose doesn't make sense any longer. We should just remove the whole file.

Reply:
The dev mode script here starts only the dse, so data api can be started in local for dev.

Reply:
Got rid of the dev mode script.